external-crates/move/crates/move-compiler/tests/move_2024/ide_mode/dot_incomplete.ide.exp

```diff
@@ -1,8 +1,38 @@
+note[I15001]: IDE autocomplete
+   ┌─ tests/move_2024/ide_mode/dot_incomplete.move:1:1
+   │  
+ 1 │ ╭ module a::m {
+ 2 │ │ 
+ 3 │ │     public struct A has copy, drop {
+ 4 │ │         x: u64
+   · │
+18 │ │     }
+19 │ │ }
+   │ ╰─^ Possible in-scope names: 'A -> a::m::A', 'B -> a::m::B', 'Option -> std::option::Option', 'Self -> a::m', 'option -> std::option', 'vector -> std::vector', 'K -> 0x19', 'M -> 0x40', 'a -> 0x44', 'b -> 0x45', 'k -> 0x19', 'std -> 0x1', or 'sui -> 0x2'
+
+note[I15001]: IDE autocomplete
+  ┌─ tests/move_2024/ide_mode/dot_incomplete.move:8:12
+  │
+8 │         a: A
+  │            ^ Possible in-scope names: 'A -> a::m::A', 'B -> a::m::B', 'Option -> std::option::Option', or 'foo -> a::m::foo'
```

Comment:
For types we should add primitives to auto-completion list

Reply:
Can we do this as another diff? This is primarily to generate the in-scope names, and we can do that on the IDE side.